packages/examples/cvat/exchange-oracle/src/modules/api.py

```diff
@@ -22,13 +22,16 @@ def jl_webhook(
     jl_webhook: JLWebhook,
     human_signature: Union[str, None] = Header(default=None),
 ):
-    validate_signature(human_signature)
-    validate_escrow(jl_webhook.chain_id, jl_webhook.escrow_address)
+    try:
+        validate_signature(human_signature)
+        validate_escrow(jl_webhook.chain_id, jl_webhook.escrow_address)
 
-    with SessionLocal.begin() as session:
-        webhook_id = create_webhook(session, jl_webhook, human_signature)
+        with SessionLocal.begin() as session:
+            webhook_id = create_webhook(session, jl_webhook, human_signature)
 
-    return JLWebhookResponse(id=webhook_id)
+        return JLWebhookResponse(id=webhook_id)
+    except Exception as e:
+        raise HTTPException(status_code=400, detail=str(e))
```

Comment:
I believe you added this try-except block to return 400 in case something went wrong during escrow validation.
But catching `Exception` here every exception will cause a 400 response and expose inner details of the error outside (for example, if database will fail to create a new entity. In this case we need to send back 500 error without any details). I suggest you here use `ValueError` instead of `Exception`.